testproject/Assets/Tests/Runtime/OnNetworkSpawnExceptionTests.cs

```diff
@@ -48,6 +48,7 @@ public class OnNetworkSpawnExceptionTests : BaseMultiInstanceTest
         [UnityTest]
         public IEnumerator WhenOnNetworkSpawnThrowsException_FutureOnNetworkSpawnsAreNotPrevented()
         {
+            LogAssert.ignoreFailingMessages = true;
```

Comment:
IIRC, there was something like LogAssert.Expect or something. wouldn't that work here?